RocketChat/Rocket.Chat
# [FIX] Notification Preference Save Button Update

IMPROVE



## Proposed changes (including videos or screenshots)


In Notification Preferences, when you check the option Mute and after that uncheck again, the save button disable and never eneble more when you check and uncheck de Mute checkbox. The changes come to solve that.



## Issue(s)

Issue: #27512
## Steps to test or reproduce

Go to one conversation -> notification preference bell(top right) -> check and uncheck mute @all and @here mentions

## Further comments
